### DISCUSSION TITLE
Property [$numberOfPaginatorsRendered] not found on component: [company-table]

### DISCUSSION THREAD

**Discussion by roomdada** [luckykenlin/livewire-tables#17] [Q&A] [answered]
Hello everyone i need your help to fix this bug.
![image](https://github.com/luckykenlin/livewire-tables/assets/61591337/0f6fce74-3264-40a8-8ee3-10b95d0edfbe)

**Comment by luckykenlin** [comment, marked as answer]
4.0.1 will fix this
